fighthealthinsurance/fax_actor.py

```
@@ -121,9 +125,12 @@ def _update_fax_for_sent(self, fax, fax_success):
         print(f"Checking if we should notify user of result {fax_success}")
         if fax.professional:
             print(f"Professional fax, no need to notify user -- updating appeal")
-            appeal = fax.for_appeal.get()
-            appeal.sent = fax_success
-            appeal.save()
+            try:
+                appeal = fax.for_appeal.get()
+                appeal.sent = fax_success
+                appeal.save()
+            except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)